### DISCUSSION TITLE
How to fix Log4J

### DISCUSSION THREAD

**Discussion by drakeerv** [itzg/docker-minecraft-server#1218] [Q&A] [answered]
Idk if I should be worrying about this but is there a way to fix Log4J on an already created docker container? Thanks.

**Comment by itzg** [comment, marked as answer]
Make sure you have pulled the latest image, restart the container, and known mitigations will be applied.

**Reply by jurehbrezveze** [reply]
Hi, pulled the latest image, but the log4j api (config/libraries/org/apache/logging/log4j/log4j-api) is still showing version 2.14.1. Just to be super sure, is this expected? Was log4j patched without upgrading to 2.17.1?

Thanks.

**Reply by itzg** [reply]
The mitigations provided here are applied at startup

https://www.minecraft.net/en-us/article/important-message--security-vulnerability-java-edition

**Reply by loprima-l** [reply]
I just have a question about Mohist 1.12.2-320, I don't see the fixing file in the server dir, where is it stored ?
